### PR TITLE
Add Statusfy v1.4.0

### PR DESCRIPTION
**Pull request by andreis**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [ ✔️ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ✔️ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ✔️ ] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [ ✔️ ] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ✔️ ] `brew cask install {{cask_file}}` worked successfully.
- [ ✔️ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ✔️ ] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [ ✔️ ] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

### CODE REVIEW

File: Casks/statusfy.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'statusfy' do
+  version '1.4.0'
+  sha256 '7fcd2bf27000bec59ef76343621fc4c551c68dd5ab338742ca824b2c5fe0771b'
+
+  url 'https://github.com/paulyoung/Statusfy/releases/download/1.4.0/Statusfy.zip'
```
Comment (vitorgalvao) [inline Casks/statusfy.rb]: `url "https://github.com/paulyoung/Statusfy/releases/download/#{version}/Statusfy.zip"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/statusfy.rb (hunk 1 above)
Comment (andreis) [inline Casks/statusfy.rb]: That's what I did initially, but `brew cask style` complained. Will change

**Review by vitorgalvao — APPROVED** [review]

File: Casks/statusfy.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'statusfy' do
+  version '1.4.0'
+  sha256 '7fcd2bf27000bec59ef76343621fc4c551c68dd5ab338742ca824b2c5fe0771b'
+
+  url "https://github.com/paulyoung/Statusfy/releases/download/#{version}/Statusfy.zip"
+  appcast 'https://github.com/fitztrev/shuttle/releases.atom',
```
Comment (vitorgalvao) [inline Casks/statusfy.rb]: Wrong appcast.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

File: Casks/statusfy.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'statusfy' do
+  version '1.4.0'
+  sha256 '7fcd2bf27000bec59ef76343621fc4c551c68dd5ab338742ca824b2c5fe0771b'
+
+  url "https://github.com/paulyoung/Statusfy/releases/download/#{version}/Statusfy.zip"
+  appcast 'https://github.com/paulyoung/Statusfy/releases.atom',
+          checkpoint: '0c4e6266973191bc5cabe69bebae32d2698b007e1907af627694938899d87dac'
+  name 'Shuttle'
```
Comment (victorpopkov) [inline Casks/statusfy.rb]: Wrong name.

**Review by victorpopkov — CHANGES_REQUESTED** [review]

**Review by victorpopkov — APPROVED** [review]

**Comment by victorpopkov** [comment]
Thank you for all the changes and your contribution!

**Pull request merged**